src/motion/features/layout/Animate.tsx

```diff
@@ -210,19 +245,18 @@ class Animate extends React.Component<AnimateProps> {
 
             // Tween the axis and update the visualElement with the latest values
             tweenAxis(frameTarget, origin, target, p)
+
             visualElement.setProjectionTargetAxis(
                 axis,
                 frameTarget.min,
-                frameTarget.max
+                frameTarget.max,
+                isRelative
             )
         }
 
         // Synchronously run a frame to ensure there's no flash of the uncorrected bounding box.
         frame()
 
-        // Ensure that the layout delta is updated for this frame.
-        visualElement.updateLayoutProjection()
```

Comment:
It's not entirely clear to me why this isn't needed anymore?

Reply:
Is this because all children no longer have a delta since they're always relatively resolved?

Reply:
The `flushSync` calls essentially do this now. This was a bit brute force anyway - it was a sync call that would update the entire tree below it. For each element calling it. Instead we do this at the end of the batcher, just once for the whole tree.

Reply:
Oh awesome

Reply:
Oh and it was incompatible with the two-pass anyway, as it would schedule the second pass for the next frame step. Which is why we manually flush these out via framesync now. It ensures all trees first get relative boxes updated and then all projections updated, and then rendered, just once